spec/ruby/core/thread/inline_rescue_spec.rb

```diff
@@ -0,0 +1,9 @@
+require File.expand_path('../../../shared/kernel/raise', __FILE__)
+
+describe "inline rescue" do
```

Comment:
Since at its core this spec is testing if `||=` works correctly with a `rescue`, I'd say this description should be something along the lines of "Optional assignments".